frontend/src/components/sidebar/thread-list.tsx

```diff
@@ -33,7 +32,7 @@ async function getThreads(): Promise<BThread[]> {
 
 export async function ThreadList() {
   const threads = await getThreads();
-
+  console.log(threads);
```

Comment:
again, let's remove this one (and all the other `console.log` if there are more)